build-tools/packages/build-cli/docs/codeCoverageDetails.md

```diff
@@ -91,3 +91,16 @@ The code coverage PR checks will fail in the following cases:
 - If the code coverage(line and branch) for a newly added package is < 50%.
 
 The enforcement code is in the function `isCodeCoverageCriteriaPassed`.
+
+### What to do if the code coverage check fails:
```

Comment:
I think this would be better inlined in the PR comment. The flub command on its own is used by the pipelines but not inherently tied to them or what happens in them, so having these docs here feels off to me. These docs are about builds and process, and the command is just a tool leveraged by them.

Reply:
Done.